templates/partials/snipcart_subscription_item.html.twig

```diff
@@ -16,17 +16,23 @@
         <p>
             <a href="#"
                 class="button snipcart-add-plan"
```

Comment:
https://docs.snipcart.com/v2/configuration/recurring-and-subscriptions

It needs to be <button class="snipcart-add-item"